Arm/Insts/DPSFP/Advanced_simd_three_different.lean

```diff
@@ -37,8 +37,8 @@ def pmull_op (e : Nat) (esize : Nat) (elements : Nat) (x : BitVec n)
   if h₀ : elements <= e then
     result
   else
-    let element1 := elem_get x e esize H
-    let element2 := elem_get y e esize H
+    let element1 := elem_get x e esize
+    let element2 := elem_get y e esize
```

Comment:
Can we drop the `H` parameter altogether here? It only seems to be explicitly used in the recursive call to `pmull_op`. It might be picked up by one of the `omega` calls as well. Still, worth a try

Reply:
Yes, it is picked up by one of the omega call (in `h_2`), which in turn is used in `elem_set`.